wiki/en/en-Adminstration.md

```diff
@@ -5,4 +5,90 @@ lang: "en"
 permalink: "/wiki/Adminstration"
 ---
 
+This document describes internal processes and guidelines.
+It is public for transparency.
+There currently is no need to translate this page.
 
+# Releases
+## Versioning
+We use [Semantic Versioning 2.0.0](https://semver.org/).
+This means:
+- We bump the major version (e.g. Jamulus 3.21.0 to 4.0.0) only in case of backwards-incompatible or large architectural changes.
+- We bump the minor version (e.g. Jamulus 3.9.0 to 3.10.0) for new features and backwards-compatible improvements.
+- We bump the patch level version (e.g. 3.7.0 to 3.7.1) for small bugfixes. Examples include:
+  - Bug fixes for issues introduced with by the last release.
+  - Addition of translations which were too late to be included into the last release.
+
+Contrary to Semantic Versioning, we currently do not place a hyphen between the version and pre-release suffixes.
+
+We use the following suffixes for pre-releases:
+- `dev` to denote builds from git. Example: `3.7.0dev-123456` is an inofficial release built from code on top of the final 3.7.0 release.
+- `beta` to denote test builds for new releases. Example: `3.8.0beta1` is the first official test build before 3.8.0 final is tagged. Betas are still subject to change.
+- `rc` to denote release candidates. Example: `3.8.0rc1`is the first release candidate before 3.8.0 final is tagged. Release candidates must not receive any code changes. If they do (e.g. due to critical bugfixes), another rc has to be tagged.
+
```

Comment:
Probably we could also add nightly here?

Reply:
`inofficial` should be `unofficial`, too.

Reply:
> Probably we could also add nightly here?

Is it OK the way I just added it?

Reply:
Mmm,

> to denote a custom builds from git.

vs
> to denote a builds from git.

is a bit vague.  I think it needs to clearly state "to denote an official build from the jamulussoftware/jamulus master branch, produced by the project's CI system on Github" or something equally explicit.  (And correct, if I got that wrong.)

Reply:
Hrm, after some more thinking about this, I'm not sure how to follow up. I think this depends on the perspective:

1. Document how we use suffixes (this is my primary goal)
2. Document if and how others may use such suffixes (this is currently not a goal of mine as part of this PR)
3. Change how autobuild works (especially regarding version handling for autobuilds from branches/PRs). While it may sound simple to get the branch (and integrate it), the branch name my not be unique at all. My fork could have a totally different master branch than jamulussoftware/jamulus. So we would end up having to integrate the git repo name as well, etc. I think this quickly becomes complicated and I wonder if this really needs a technical solution.

What I'll do now as part of this PR, I'll clarify the scope of the versioning scheme (1) and will add a statement that forks may not re-use the Jamulus name & versions in order to avoid version confusion. This is only a side note, in my opinion, as the document is targeted at us. There is a separate CONTRIBUTE.md PR which @gilgongo is working on which might be a better fit for that?

Could you open a separate issue if you see any specific needs for changes in the autobuild logic?

I have pushed an update which (hopefully) incorporates your feedback. Could you check if this is sufficient?

- I added a distinction between `custom builds from git` (dev) and `deliberate snapshots from the official jamulussoftware/jamulus git` (devNightly).
- I have added a statement about (not having) release versions in forks